lib/src/components/NxList/__tests__/NxList.test.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { ReactElement } from 'react';
+import { NxH3, NxLoadError, NxLoadingSpinner } from '../../..';
+import NxList from '../NxList';
+import { NxListProps } from '../types';
+import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { act } from 'react-dom/test-utils';
+import { mount, ReactWrapper } from 'enzyme';
+
+describe('NxList', function() {
+
+  let mountContainers: HTMLElement[] = [];
+
+  function getMountContainer() {
+    const newContainer = document.createElement('div');
+    mountContainers.push(newContainer);
+    document.body.append(newContainer);
+
+    return newContainer;
+  }
+
+  // create a mounted wrapper that is attached to the document, and deal with the timing complexities
+  // triggered by NxList's MutationObserver usage
+  async function mountAttached(jsx: ReactElement) {
+    let retval: ReactWrapper;
+    await act(async () => {
+      retval = mount(jsx, { attachTo: getMountContainer() });
+    });
+    retval!.update();
+
+    return retval!;
+  }
+
+  afterEach(function() {
+    mountContainers.forEach(container => document.body.removeChild(container));
+    mountContainers = [];
+  });
+
+  const minimalProps: NxListProps = {};
+  const getShallow = getShallowComponent(NxList, minimalProps);
+  // const getMounted = getMountedComponent(NxList, minimalProps);
+
+  it('renders a list', function() {
+    const nxList = getShallow();
+    expect(nxList).toMatchSelector('.nx-list');
+  });
+
+  it('renders the classNames given to it', function() {
+    const extendedProps: Partial<NxListProps> = {
+      className: 'test-classname ufo'
+    };
+    const nxList = getShallow(extendedProps);
+    expect(nxList).toMatchSelector('.nx-list.test-classname.ufo');
+  });
+
+  it('passes props to the parent component and renders bulleted list', function() {
```

Comment:
What does "passes props to the parent component" mean here?

Reply:
> What does "passes props to the parent component" mean here?

I apologize if that was unclear. I have re-written the description and tests for it.
ca501e674c08ea34c34330b9df74658af5804120